apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/McsTransformerIT.java

```diff
@@ -374,9 +374,7 @@ private static Arguments newNonFirstClaimCase() {
             .setIdrClmHdIcn(NEW_CLAIM_ICN)
             .setMcsDiagnosisCodes(
                 0,
-                defaultClaim
-                    .getMcsDiagnosisCodes(0)
-                    .toBuilder()
+                defaultClaim.getMcsDiagnosisCodes(0).toBuilder()
```

Comment:
Really having a hard time parsing when it decides something should be on an indented new line or not with these chained calls.